src/kaleidoscope/driver/storage/NoStorage.h

```diff
@@ -0,0 +1,31 @@
+/* -*- mode: c++ -*-
+ * NoStorage -- Dummy Storage hardware driver for Kaleidoscope
```

Comment:
NoOpStorage?

Really, the more of these no-op classes, I see, the more I wonder if each should just be named 'None'

Reply:
OOooooh!

`kaleidoscope::driver::storage::None`, I like that. Thanks!